### PR TITLE
Convert cookie banner to use design system component

### PR DESCRIPTION
**Pull request by kr8n3r**
## What 

Convert existing cookie banner be inline with the design system component. To fix this issue:

https://trello.com/c/SKYjfg19/692-change-cookie-banner-to-use-design-system-component

Changes include update to the HTML markup, in line with https://design-system.service.gov.uk/components/cookie-banner/ as well as associated JavaScript changes to restore functionality.

Content is a combination of our existing banner and the upstream default

### Before

Screenshot 2024-04-12 at 14 53 47
Screenshot 2024-04-12 at 14 53 57

### After

Screenshot 2024-04-19 at 11 10 22
Screenshot 2024-04-19 at 11 10 33

### CODE REVIEW

File: app/assets/javascripts/cookieMessage.js [hunk 1]
```diff
@@ -79,20 +79,28 @@ window.GOVUK.NotifyModules = window.GOVUK.NotifyModules || {};
...
+    // Set tabindex to -1 to make the confirmation banner focusable with JavaScript
+    if (!target.getAttribute('tabindex')) {
+      target.setAttribute('tabindex', '-1');
+
+      target.addEventListener('blur', () => {
+        target.removeAttribute('tabindex');
+      });
+    }
+  
+    target.focus();
```
Comment (tombye) [inline app/assets/javascripts/cookieMessage.js:103]: This is much better moved to here 👍🏻.

**Review by tombye — CHANGES_REQUESTED** [review]
This is great and nice to see us now able to delete more custom code.

Only change I think is needed is to move the remaining CSS in `app/assets/stylesheets/components/cookie-message.scss` to `app/assets/stylesheets/govuk-frontend/overrides.scss` to put it with the others.

**Comment by kr8n3r** [comment]
> This is great and nice to see us now able to delete more custom code.
> 
> Only change I think is needed is to move the remaining CSS in `app/assets/stylesheets/components/cookie-message.scss` to `app/assets/stylesheets/govuk-frontend/overrides.scss` to put it with the others.

done

**Review by tombye — APPROVED** [review]
Looks perfect now 👍🏻

**Review by idavidmcdonald — APPROVED** [review]
Nice, I've reviewed the 2 commits since Tom's latest approval and they look all fine to me (especially given the tests are passing). I'm not an expert on this stuff but given that, I think I'm Ok to approve.

**Pull request merged**